docs/Functional.md

```diff
@@ -1,51 +1,121 @@
 # Functional Specification
 
-Clement Caton Last update : 14/11/22
+Clement Caton Last update : 18/11/22
 
 ## Summary
 
 - [Summary](#summary)
 - [Requirements](#requirements)
+- [Personae](#personae)
 - [Objectives](#objectives)
 - [User documentation](#user-documentation)
 - [Test plan](#test-plan)
+- [Marketing plan](#marketing-plan)
 - [The final product](#the-final-product)
 
 
 ## Requirements.
-<!--This is a formal statement of what the product planners informed by their knowledge of the marketplace and specific input from existing or potential customers believe is needed for a new product or a new version of an existing product. Requirements are usually expressed in terms of narrative statements and in a relatively general way.-->
-
-
 
 ### Functional requirements
 
 - The service will be able to ask the client to accept the use of his location.
 - The service will be able to take a postal address and return a time for Santa to arrive.
 - The service will be able to take a coordinate and return a time for Santa to arrive.
+- The service will take into consideration the [Equation of time](https://en.wikipedia.org/wiki/Equation_of_time).
 
 ### Non-functional requirements
 
 - The service will be able to handle 10k requests per minute.
 - The service will not call an external API.
+- The service will be scalable thanks to kubernetes
+
+## Personae
+
+<img src="./thispersondoesnotexist/dev2.jpg" alt="drawing" width="200"/><br>
+Name: Marc<br>
+Age: 22<br>
+Education: Master's degree in computer science<br>
+Background: Student<br>
+Skills: Good knowledge of computer science<br>
+Geography: France<br>
+Platform: Web Desktop<br>
+Goal: Find out when Santa will arrive<br>
+Motivation: I want to know why this website exists<br>
+Pain points or frustrations: I don't like to sign up to a service<br>
+
+<img src="./thispersondoesnotexist/kid.jpg" alt="drawing" width="200"/><br>
+Name: Lisa<br>
+Age: 8<br>
+Education: None<br>
+Background: Child<br>
+Skills: Knows of the existance of Santa<br>
+Geography: UK<br>
+Platform: Web Ipad<br>
+Goal: Find out when Santa will arrive<br>
+Motivation: Embush Santa to prank him<br>
+Pain points or frustrations: I don't like hard interfaces<br>
```

Comment:
What is considered as a bad interface ?

Reply:
unrelevent, it's hard, not bad
hard as in bard to navigate